pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -114,7 +114,8 @@ func (p *provider) getConfig(s runtime.RawExtension) (*Config, *providerconfig.C
 	}
 	c.DomainName, err = p.configVarResolver.GetConfigVarStringValueOrEnv(rawConfig.DomainName, "OS_DOMAIN_NAME")
 	if err != nil {
-		return nil, nil, nil, fmt.Errorf("failed to get the value of \"domainName\" field, error = %v", err)
+		glog.V(0).Infof("[warning] could not get the value of the \"domainName\" field, this means instance %s",
```

Comment:
I am not entirely sure if this is the best way to handle that case but didn't come up with a better one. Ideally we would log that as event on the machine, however that would require passing an `EventRecorder` into the cloudprovides.

WDYT? Maybe other ideas?

Reply:
Not for sure, `v2` endpoints often have a `/v2.0` prefix but I do not think that is strictly required

Reply:
So why then logging it at all?
It will fail during creation - which is fine. I would rely on the openstack error handling here.